opentech/apply/review/views.py

```diff
@@ -30,32 +34,48 @@ def get_context_data(self, **kwargs):
         for review in reviews:
             reviews_dict[review.author.pk] = review
 
+        # Get all the authors of opinions, these authors should not show up in the 'xxx_not_reviewed' lists
+        opinion_authors = set(User.objects.filter(
+            pk__in=ReviewOpinion.objects.filter(review__submission=self.object).values('author__pk')))
```

Comment:
`pk__in=self.object.reviews.opinions.values('author')` maybe?

Would involve an extra method on the review queryset.

Reply:
good idea